yarn.lock

```diff
@@ -6735,6 +6747,11 @@ mkdirp@^0.5.0, mkdirp@^0.5.1, mkdirp@~0.5.0, mkdirp@~0.5.1:
   dependencies:
     minimist "0.0.8"
 
+mobx@^5.9.4:
```

Comment:
are we using mobx still?

Reply:
no, those are leftover from a previous commit with mobx